src/data/ACTIONS/root/SAM.ts

```diff
@@ -82,7 +82,20 @@ export const SAM = ensureActions({
 		icon: 'https://xivapi.com/i/003000/003158.png',
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
-		potency: 320,
+		// These are 6.0 values updated in the 6.1 layer.
```

Comment:
Unnecessary comment, since they'll likely be updated in future layers too and nobody should have to maintain a data comment like this.

Reply:
Fixed